cmd/webhook-manager/app/util.go

```diff
@@ -17,93 +17,104 @@ limitations under the License.
 package app
 
 import (
+	"bytes"
 	"context"
 	"crypto/tls"
 	"crypto/x509"
-	"regexp"
-	"strings"
+	"fmt"
+	"time"
 
 	v1 "k8s.io/api/admissionregistration/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+
```

Comment:
remove the line

Reply:
Thanks for your suggestion. I deleted this.